age-plugin/src/identity.rs

```diff
@@ -276,7 +276,7 @@ pub(crate) fn run_v1<P: IdentityPluginV1>(mut plugin: P) -> io::Result<()> {
         });
 
         let stanzas = stanzas.and_then(|recipient_stanzas| {
-            let mut stanzas: Vec<Vec<Stanza>> = Vec::new();
+            let mut stanzas: Vec<Vec<Stanza>> = Vec::with_capacity(recipient_stanzas.len());
```

Comment:
This reliably over-allocates. `stanzas` will have length based on the number of files, not the total number of recipient stanzas. Please remove this change from the PR.

As a side-note, for the main age client implementations, only a single file is encrypted at a time, so `stanzas` will have length 1. In that case, the previous code was allocating `stanzas` only once (because `Vec::new()` and `vec![]` do not allocate).

Reply:
```suggestion
            let mut stanzas: Vec<Vec<Stanza>> = Vec::new();
```